pkg/abap/build/connector.go

```diff
@@ -174,3 +176,39 @@ func (conn Connector) UploadSarFile(appendum string, sarFile []byte) error {
 	defer response.Body.Close()
 	return nil
 }
+
+// UploadSarFileInChunks : upload *.sar file in chunks
+func (conn Connector) UploadSarFileInChunks(appendum string, fileName string, sarFile []byte) error {
```

Comment:
Where is the test for this function?

Reply:
it is tested by the command unit tests - e.g. cmd\abapAddonAssemblyKitRegisterPackages_test.go